packages/vuetify/src/components/VTreeview/VTreeviewNode.ts

```diff
@@ -213,11 +218,14 @@ const VTreeviewNode = baseMixins.extend<options>().extend({
         class: {
           'v-treeview-node__toggle--open': this.isOpen,
           'v-treeview-node__toggle--loading': this.isLoading,
+          'v-treeview-node__toggle--disabled': this.disabled && this.selectionType === 'leaf',
```

Comment:
`this.disabled && this.selectionType === 'leaf'` is duplicated on line 221, 226, 293